modules/cnv-about-node-network-state.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-node-network-state.adoc
+
+[id="cnv-about-node-network-state_{context}"]
+= About node network state
```

Comment:
I find this section a little confusing in the rendered docs. The page is about observing of the state. This title suggest it is about the Node Network State object (at least to me), but then you have the introduction to the whole kubernetes-nmstate. Can we make sure that this part of the docs is distinguishable introduction? Either have a separate page OR have this intro on both pages and make it obvious that it is not a part of the "tutorial" that is described below.

Reply:
Have updated the heading (and id) to be about nmstate, and included it as the opening for the updating-node-network assembly as well.